### ISSUE TITLE
Deno bundle bad output - _handlebarsDewDew

### ISSUE THREAD

**Issue by burkeholland** [alosaur/handlebars#8]
Deno Version: 1.5.3
TypeScript: 4.1

The ouptut of `Deno build` throws an error when importing Handlebars. The error is....

```bash
error: Uncaught ReferenceError: _handlebarsDewDew is not defined
    var handlebars = _handlebarsDewDew()['default'];
```

### Steps to reproduce

1. Clone [https://github.com/burkeholland/oak-express](https://github.com/burkeholland/oak-express)
1. Bundle the project with the following command....
   ```bash
   deno bundle --unstable app.ts app.build.js
   ```
1. Run the bundled file
   ```bash
   deno run -A --unstable app.build.js
   ```

### What should happen?
The program should run on port 3000

### What happens instead?
The console throws the following error...

```bash
error: Uncaught ReferenceError: _handlebarsDewDew is not defined
    var handlebars = _handlebarsDewDew()['default'];
```

**Comment by irustm** [comment]
Thank you for this issue.

I think it is a Deno problem because it does not support `import as`, which is in jspm. 
https://dev.jspm.io/npm:handlebars@4.7.6/lib/index.dew.js

```ts
import { dew as _handlebarsDewDew } from "../dist/cjs/handlebars.dew.js";
```

https://github.com/denoland/deno/issues/8285 and https://github.com/denoland/deno/issues/8314

**Comment by burkeholland** [comment]
That looks correct. Thank you for the quick response. We can close this issue.

**Comment by kdy1** [comment]
Is this fixed with deno 1.5.4?
